src/Symfony/Bridge/Twig/Translation/TwigExtractor.php

```diff
@@ -77,8 +77,22 @@ protected function extractTemplate(string $template, MessageCatalogue $catalogue
 
         $this->twig->parse($this->twig->tokenize(new Source($template, '')));
 
-        foreach ($visitor->getMessages() as $message) {
-            $catalogue->set(trim($message[0]), $this->prefix.trim($message[0]), $message[1] ?: $this->defaultDomain);
+        foreach ($visitor->getMessages() as $extractedMessage) {
+            $message = trim($extractedMessage[0]);
+            $domain = $extractedMessage[1] ?: $this->defaultDomain;
+
+            $catalogue->set($message, $this->prefix.trim($extractedMessage[0]), $domain);
+
+            if (!empty($extractedMessage[2])) {
+                $metadata = $catalogue->getMetadata($message, $domain);
+
+                $metadata['notes'][] = [
+                    'category' => 'symfony-extractor-variables',
+                    'content' => 'Available variables: '.implode(', ', $extractedMessage[2]),
```

Comment:
Is it ok to use a hard-coded string here? I thought of using a translated string (e.g. to the destination locale passed to `translator:update`), but it may be an overkill...

Reply:
I think it is fine. 

We can shorten it by just doing "Variables:"